client/js/keybinds.js

```diff
@@ -156,6 +169,13 @@ const ignoredKeys = {
 };
 
 document.addEventListener("keydown", (e) => {
+	// Allow navigating back to the previous page when on the help screen.
+	if (e.which === 27 && router.currentRoute.name === "Help" && previousRoute) {
```

Comment:
iirc, `e.which` isn't recommended. Can we use `e.key === "Escape"`? It should have more support for non-standard keyboards.

Reply:
👍 I used which because the rest of the file also seemed to use it. This makes more sense, fixed.